src/common/mmo.hpp

```diff
@@ -89,7 +89,7 @@ typedef uint32 t_itemid;
 #endif
 #define MAX_FAME 1000000000 ///Max fame points
 #define MAX_CART 100 ///Maximum item in cart
-#define MAX_SKILL 1481 ///Maximum skill can be hold by Player, Homunculus, & Mercenary (skill list) AND skill_db limit
+#define MAX_SKILL 1486 ///Maximum skill can be hold by Player, Homunculus, & Mercenary (skill list) AND skill_db limit
```

Comment:
```suggestion
#define MAX_SKILL 1482 ///Maximum skill can be hold by Player, Homunculus, & Mercenary (skill list) AND skill_db limit
```

Reply:
This is intentional to prevent conflicts between the PRs.
It does not really hurt to increase it more than needed.